### ISSUE TITLE
Pixel 6a: Heat issue

### ISSUE THREAD

**Issue by LotusIodid** [GrapheneOS/os-issue-tracker#2001]
When I start a Wi-Fi hotspot with enable 5G and use this for around 25 min, the phone turns off due to heat. The Wi-Fi usage is very moderate, but constant.

Roughly 15 min earlier, I got a warning: “Phone is getting warm”. After the reboot, I do get the message: “Phone turned off due to heat.”.

This issue does not occur with the original Android OS. The issue is reproducible among two different Pixel 6a. The environment temperature is ~21 °C.

With Android OS, the phone still gets warm, but it does not turn off.

**Comment by LotusIodid** [comment]
The problem does not occur with Ethernet tethering.

**Comment by SB13X** [comment]
I use my 6a regularly as a hotspot, plugged in to the wall too. I have only ever experienced a heat warning when using the device for something intensive such as a video call at the same time. Beyond that I can have it on hotspot for approx 9 hours without issue. Fair enough I tend to run LTE only mode if that helps.

**Comment by thestinger** [comment]
Doesn't sound like a GrapheneOS issue bu rather a configuration / usage difference ultimately leading to uncovering a hardware issue.

**Comment by LotusIodid** [comment]
@Metr0pl3x LTE only mode really helps, the device doesn't get only slightly warm.

@thestinger The issue does not appear in Android with the default config, and appears in GrapheneOS in the default config on the exactly same devices. I have tested and reproduced this on two devices with both operating systems.

I did conduct these tests:
- (Device 1, GrapheneOS) -> Power off.
- (Device 1, Android) -> Gets very warm, but doesn't power off.
- (Device 2, GrapheneOS) -> Power off.
- (Device 2, Android) -> Gets very warm, but doesn't power off.
- (Device 1, GrapheneOS, LTE Only) -> Gets only slightly warm.
I repeated the tests with Device 2 twice.

The usage was not exactly the same, but comparable. (Web browsing, ping, ssh, two speed tests, No streaming)

It seems to be that google was able to do something about thermal performance in an update in December last year.
https://support.google.com/pixelphone/thread/191508547
